src/digitalIdentity-backend/src/main/java/didentity/amos/digitalIdentity/services/LissiApiService.java

```diff
@@ -101,10 +100,14 @@ public ResponseEntity<String> provideExistingSchemas(String activeState, String
         return handleResponse(response);
     }
 
-    public ResponseEntity<String> createCredentialDefinition(String alias, String comment, String imageUri,
-            String schemaId,
-            File file, String revocable) {
+    @SuppressWarnings("unchecked") // TODO: if someone wants to bother with generic arrays, feel free :)
+    public ResponseEntity<String> createCredentialDefinition(String alias, String comment, String imageUri, String schemaId,
+                                                             File file, String revocable) {
```

Comment:
This is not adapted to the LissiApiService. It has to be:

public ResponseEntity<String> createCredentialDefinition(String alias, String comment, String imageUri,
            String schemaId, File file, String revocable) {